File: official/docs/java/insurances/RetrieveInsurance.java
package insurances;

import com.easypost.EasyPost;
import com.easypost.exception.EasyPostException;
import com.easypost.model.Insurance;

public class RetrieveInsurance {
    public static void main(String[] args) throws EasyPostException {
        EasyPost.apiKey = System.getenv("EASYPOST_API_KEY");

        Insurance insurance = Insurance.retrieve("ins_...");

        System.out.println(insurance);
    }
}

File: official/docs/csharp/current/insurances/retrieve-insurance.cs
using System;
using System.Collections.Generic;
using System.Threading.Tasks;
using Newtonsoft.Json;
using EasyPost;

namespace EasyPostExamples
{
    public class Examples
    {
        public static async Task Main()
        {
            string apiKey = Environment.GetEnvironmentVariable("EASYPOST_API_KEY")!;

            var client = EasyPost.Client(apiKey);

            Insurance insurance = await client.Insurance.Retrieve("ins_...");

            Console.WriteLine(JsonConvert.SerializeObject(insurance, Formatting.Indented));
        }
    }
}
